tests/webview/ui/test_content.py

```diff
@@ -816,5 +814,5 @@ def test_books_this_page_in_list(webview_base_url, selenium, page_id):
     # THEN check if the books list exists and on the left
     assert content.books_containing.book_list
 
-    window_width = utils.get_window_size(content, 'width')
-    assert content.location['x'] < window_width/2
+    window_width = content.get_window_size('width')
```

Comment:
move to under the `WHEN` clause and edit WHEN comment.

`# WHEN we load the page of the chapter and we have the width of the window`